### ISSUE TITLE
Dropdown menu could not support mobile

### ISSUE THREAD

**Issue by rockkratos** [zurb/foundation-sites#9293]
Hi, 

I found the dropdown menu plugin could not support mobile browser. 
When touch the submenu item, the click event has not triggered yet and the item will disappear. 
And the under DOM element will be touched. 

I double checked the official site demo, also has this issue. 

Is it a bug?

**Comment by rbayliss** [comment]
Yeah, I'm experiencing this too.  I think it's happening as a result of #9131.  The 'click' event is being fired when you touch any of the submenu items.  You can temporarily work around it by adding a data-close-on-click-inside="false" attribute to the dropdown menu.

**Comment by rockkratos** [comment]
Thanks for your reply, it works, that's fine. Thank you.

**Comment by memoht** [comment]
Thank you @rbayliss for the work around. I just completed upgrading from `5.5.x` to `6.2.4` and this bug reared it's ugly head. I just confirmed the workaround works on my iOS 10 device. Looking forward to `6.2.5` soon I hope.

**Comment by kball** [comment]
6.3 is our next release coming in about a month, but we definitely will get this fixed for that. @sujayjaju as the implementor for #9131 can you take a look at this?

**Comment by kball** [comment]
@rbayliss @memoht @rockkratos a fix for this has just been submitted as a pr: #9338.  Can you try this out and see if it works for you?

**Comment by memoht** [comment]
I'd be happy to try this but unsure how to pull in this version since I am using Bower and linking to the dist folder. What would I put in my `bower.json` file under dependencies to use this version instead of `"foundation-sites": "6.2.4"`

**Comment by memoht** [comment]
Ok. That didn't work out for me. The `foundation.js` in the `dist` folder didn't have this code. I tried to edit to match the code changed in the commit and the links started to work, but the whole dropdown menu part broke.

**Comment by kball** [comment]
Ah bother, the dist build is only typically rebuilt on release, and if you plug in the change directly it won't work because of the es2016 base version.  I'm attaching a zipfile of dist built from the current HEAD of develop, can you try with this?
[dist.zip](https://github.com/zurb/foundation-sites/files/586321/dist.zip)

**Comment by memoht** [comment]
@kball Wolfing down some lunch and will test it out and get back to you.

**Comment by memoht** [comment]
@kball Just tested this out on my iOS devices and it appears to be working. Made sure to remove the workaround `"data-close-on-click-inside" => "false"` first. Thanks for your time and energy. Have a great day. LLAP +1

**Comment by sujayjaju** [comment]
@kball Just for my reference, can you please point me to any docs on how to create a build to test? I should be able to contribute in a better way. I am new to the setup. Thanks!
